eradiate/scenes/biosphere/_discrete.py

```diff
@@ -1272,8 +1270,10 @@ def from_file(cls, filename=None, plant=None):
         if not os.path.isfile(filename):
             raise FileNotFoundError(f"no file at {filename} found.")
 
-        if plant is None:
-            plant = Plant()
+        if canopy_element is None:
+            canopy_element = CanopyElement()
```

Comment:
This should break: `CanopyElement` is abstract. Instead, you can initialise with `LeafCloud()` or `AbstractTree()`.